clippy.toml

```diff
@@ -1,5 +0,0 @@
-disallowed-methods = [
```

Comment:
We _might_ want to keep these for the rest of the codebase and do `#[allow(clippy::disallowed_methods)]` on the calls in `src/executor.rs` (where we have a reason to use the raw `spawn` method).

Reply:
Makes sense:)